src/shared/sleep-config.c

```diff
@@ -95,7 +98,19 @@ int parse_sleep_config(const char *verb, char ***_modes, char ***_states) {
                 else
                         states = strv_new("disk", NULL);
 
-        } else
+        } else if (streq(verb, "suspend-to-hibernate")) {
+                /* minutes */
+                if (hibernate_delay)
+                        USE(modes, hibernate_delay);
+                else
+                        modes = strv_new("180", NULL);
```

Comment:
"180"?

Reply:
The thought here was 180 minutes is a reasonable default.
I guess you mean you'd like to see units in a comment?
or
Do you think it's better to move this to a conf file and if nothing is declared in conffile fail rather than hide a default in code?

Reply:
Oh, OK, now I understand. 180 minutes is OK. The default should be hardcoded in the code, but as a number. That is the general policy, so something sensible will happen even if all config files are purged.

It should be something like `180 * 60 * USEC_PER_SEC`, assuming that the type will be changed to `usec_t`.